packages/odyssey-react/src/components/Box/Source.tsx

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { forwardRef } from "react";
+import type { BoxProps } from "./Box";
+
+const Source = forwardRef<HTMLDivElement, BoxProps>(() => <></>);
+
+export { Source };
```

Comment:
what is this module use for? I'm not sure I follow how it is useful? Is this the docgen hack? I'm concerned about having `Source` as a top level part of our API from `import * from "@okta/odyssey-react"`

Reply:
Yes, this is the docgen hack. I didn't think this would get picked up automatically in the top level API, just the index file right?

Reply:
ahhh I see, I missed the fact that you changed the import within `odyssey-storybook`, i.e.

```
import { Source } from "../../../../odyssey-react/src/components/Box/Source"; 
```

Reply:
@andrewberg-okta can you please move this file into `@okta/odyssey-storybook/src/components/Box` to live alongside the Box story module as a sibling? It allows us to avoid the need for the complicated relative import from that Box story module, instead we can use `import { BoxProxy } from './proxy'` or similar. Also consider that this code has no meaning for the `@okta/odyssey-react` package only for our storybook to generate prop tables